connectionagentplugin/declarativeconnectionagent.cpp

```diff
@@ -31,8 +31,8 @@ DeclarativeConnectionAgent::DeclarativeConnectionAgent(QObject *parent)
       connManagerInterface(nullptr)
 {
     connectiondWatcher = new QDBusServiceWatcher(CONND_SERVICE,QDBusConnection::sessionBus(),
```

Comment:
as there are "X,Y" -> "X,[space]Y" fixes made elsewhere... one more to fix here

Reply:
Sure.